src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+// We have to run them with no parallelism because ServicePointManager.DefaultConnectionLimit is left as 2 in netfx and when they are running in parallel
+// it leaves too many connections open at once which causes tests to hang since they are not able to get a new HttpWebRequest connection opened.
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
I asked that in my comments.  At the moment it sounds like that's not worthwhile as, at least according to @safern's timings, parallelism isn't buying anything for this test suite, I guess because all of the meaty tests are all on the same test class.  It'd be good to address that in the future by ensuring things can be parallelized and avoiding the need for this attribute at all, which from other discussion sounds like a test bug elsewhere in the tests.